### ISSUE TITLE
Beamer transition animation

### ISSUE THREAD

**Issue by Obi8** [Cimbali/pympress#34]
Hello Guys,

i have add a transition animation for a slide in my presentation, but it doesnt works with pympress.
In an PDF Viewer(foxit) i can see the animation.

Example:
```
\documentclass{beamer}
\usepackage[english]{babel}
\usepackage[utf8]{inputenc}
\usepackage{xcolor}
\usepackage{pgfpages}

\setbeameroption{show notes on second screen = right}

\begin{document}
	\begin{frame}
		\only<1>{\includegraphics[width=0.5\textwidth]{example-image-a}}
		\only<2>{\includegraphics[width=0.5\textwidth]{example-image-golden}}
		\transdissolve<2>
	\end{frame}
\end{document}
```

**Comment by Cimbali** [comment]
This is indeed not implemented. Anyone wanting to take this on is free to do so and make a PR, I don't think I can rank this very high on my list of priorities.

**Comment by Cimbali** [comment]
It should probably be easy enough to use a [Gtk.Revealer](https://lazka.github.io/pgi-docs/#Gtk-3.0/classes/Revealer.html#Gtk.Revealer) for [fading and sliding](https://lazka.github.io/pgi-docs/#Gtk-3.0/enums.html#Gtk.RevealerTransitionType) transitions.

That's only a small subset of the [transitions support by Poppler](https://lazka.github.io/pgi-docs/#Poppler-0.18/enums.html#Poppler.PageTransitionType), but I don't know if we really need fancy animations (I'm not sure we *need* animations at all to be entirely honest).

**Comment by susobaco** [comment]
Hi, I don't think you need sophisticated transitions, but it would be nice to have at least two, a melt between two devices and a melt to black. And if you can define the transition time, it would be perfect.

**Comment by multiflexi** [comment]
What about animations? When I use animate package in LaTeX, Pympress just skips the slide.

**Comment by Cimbali** [comment]
yes @multiflexi, they are not implemented (yet?). What kind of animations do you use? Are they really necessary to your presentation?

It's doable but it requires some work and I don't have a lot of free time these days. Feel free to give feedback and/or make a PR if it is important to you.

**Comment by multiflexi** [comment]
I use it for GIFs which I convert to PNG and then play them in a loop. It is nice to have because some stuff is easier explained to students by moving picture, for example an 4 stroke engine or anything else technical. The only possible way now is multiple slides but those can't be looped or video but that is a lot of hustle and does not work reliably.

**Comment by Cimbali** [comment]
Video works pretty reliably in pymress, and is closer to gifs than animations between slides. I'd give that a shot if I were you. Gifs are easy to convert mp4 for example, it's done all the time for various websites

**Comment by multiflexi** [comment]
I have done what you suggested. The issue is that the video does not autostart and does not loop. Also mp4 videos do not work properly but I guess it is issue of VLC and VDPAU. I use WebM and OGV anyway.

**Comment by Cimbali** [comment]
Alright, let's open an issue to look at those features (loop & autostart). Maybe you can also open another one with your full configuration (including VLC) & mp4 test file, because it works fine for me

**Comment by Cimbali** [comment]
@multiflexi have a look at #72. You can now directly embed gifs and play them in pympress.
